tests/setupAfterEnv.ts

```diff
@@ -79,4 +90,51 @@ expect.extend({
           pass: true,
         };
   },
+
+  async toThrowExecaError(
+    promise: Promise<ExecaReturnValue<string>>,
+    message: string,
+    { replacements }: { replacements: { from: string | RegExp; to: string }[] },
+  ) {
+    try {
+      await promise;
+      return {
+        message: () =>
+          'Expected running the tool to fail with the given error message, but it did not.',
+        pass: false,
+      };
+    } catch (error) {
+      if (isExecaError(error)) {
+        const stderr = [
+          {
+            from: /^\s+at.+\)$/msu,
```

Comment:
Nit: A comment explaining what we expect this to match might help! I'm not familiar with the execa error output.

Reply:
I've extracted this to a constant and added some docs to explain what it is. I also realized that this should have been "stack trace" rather than "backtrace" (I've been writing too much Ruby lately, apparently): 6e94e30...ad4fe11